lib/src/exceptions/name/null_reserved_name_exception.dart

```diff
@@ -0,0 +1,28 @@
+/// Copyright (C) 2022 Intel Corporation
+/// SPDX-License-Identifier: BSD-3-Clause
+///
+/// null_reserved_name_exception.dart
+/// An exception that is thrown when reserved name is null
+///
+/// 2022 November 15
+/// Author: Yao Jing Quek <yao.jing.quek@intel.com>
+///
+import 'package:rohd/rohd.dart';
+
+/// This Exception show that reserved name eg. definitionName is NULL but
```

Comment:
show -> shows
NULL -> `null`

Put tick's (`) around variable names and values that aren't in scope.  Use brackets ([]) around in-scope variables.